java/ql/src/Security/CWE/CWE-489/WebviewDebuggingEnabled.ql

```diff
@@ -11,9 +11,9 @@
  */
 
 import java
-import semmle.code.java.security.WebviewDubuggingEnabledQuery
-import DataFlow::PathGraph
+import semmle.code.java.security.WebviewDebuggingEnabledQuery
```

Comment:
Fixing a typo in an importable file name is a breaking change. So leave the `WebviewDubuggingEnabledQuery.qll` as a deprecated redirect to the new file and add a change note.